src/Nethermind/Nethermind.Runner/Properties/launchSettings.json

```
@@ -147,6 +147,13 @@
         "ASPNETCORE_ENVIRONMENT": "Development"
       }
     },
+    "Taiko Holesky": {
+      "commandName": "Project",
+      "commandLineArgs": "-c taiko-holesky -dd %NETHERMIND_DATA_DIR%",
```

`%NETHERMIND_DATA_DIR%` has been dropped a while ago. Needs to be replaced with `.data`

```suggestion
      "commandLineArgs": "-c taiko-holesky -dd .data",
```

Could you please ensure all occurrences of it are replaced with `.data`? Otherwise, please let me know, I'll commit myself.

---

Yup, this branch is very far from merge into master, ig it's better to update this config separately